cfgmgr/buffermgr.h

```diff
@@ -0,0 +1,54 @@
+#ifndef __BUFFMGR__
+#define __BUFFMGR__
+
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "orch.h"
+
+#include <map>
+#include <string>
+
+namespace swss {
+
+#define INGRESS_LOSSLESS_PG_POOL_NAME "ingress_lossless_pool"
+#define LOSSLESS_PGS "3-4"
```

Comment:
By defining lossless pgs to 3-4 in code, do we lose the flexibility of choosing any pg as lossless?

I know that in practice we are always have been using 3-4 as lossless pgs, but losing the capability of changing is a bit concerning.

Reply:
theoretically PGs numbers can be fetched from the PORT_QOS_MAP:pfc_enable.
@lguohan, can we rely on the fact that queues specified in PORT_QOS_MAP:pfc_enable ("3,4") will always be the ones for which we need to update ingress lossless profile on speed change?

Reply:
We had a discussion, we decided to move forward with this change for now and revisit this area and remove the hard coded pg numbers in the future.